cmd/podman/create.go

```diff
@@ -505,8 +515,40 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 		"uts":  c.String("uts"),
 	}
 
-	if c.IsSet("pod") {
-		namespaces, err = configurePod(c, runtime, namespaces)
+	originalPodName := c.String("pod")
```

Comment:
Shouldn't you just ignore this whole section if "pod" is not set?

Reply:
I was thinking of 
if c.IsSet("pod") {
     if strings.HasPrefix(originalPodName, "new:") {
                    Create POD 
     } else {
               if portbindings, throw an error
    }
}